static/scss/general_styles.scss

```diff
@@ -0,0 +1,15 @@
+// General Style Rules
+
+body {
+  background-color: $page-background;
+  font-family: 'Roboto', helvetica, arial, sans-serif !important;
+  line-height: 1.4em;
+  }
+
+
+
+
+
+
+
+
```

Comment:
I think george said something about configuring sublime, but all this is something that your editor can be configured to mostly do for you.